tests/bearlib/aspects/SubTest.py

```diff
@@ -31,28 +31,28 @@ def test_tastes(self, SubAspect, SubAspect_tastes):
             taste_values = aspect.tastes
             for name, taste in SubAspect_tastes.items():
                 if not taste.languages or language in taste.languages:
-                    assert getattr(aspect, name) == taste_values[name] \
-                        == taste.default
+                    assert getattr(aspect, name) == (
```

Comment:
`(getattr(aspect, name) == (taste_values[name]) == taste.default)` and then break the line.